how-to/laravel.md

```diff
@@ -0,0 +1,653 @@
+---
+title: How to use PGroonga with Laravel
+---
+
+**# 01. Introduction**
+In this guide we will walk through building a Laravel application from scratch with PGroonga. 
+
+**# 02. Installation**
+
+**## Installing Laravel**
+> ****Note****
+> This tutorial assumes you have already set up your environment for Laravel, PostgreSQL and PGroonga
```

Comment:
Could you avoid using `> ` markup for note because note isn't a block quote?
(I understand that Markdown isn't a feature rich markup language. But we want to avoid abuse of block quote semantics.)

How about using `**NOTE**: ...` style? (`****...****` is the same as `**...**` in Markdown.)

```suggestion
**NOTE**: This tutorial assumes you have already set up your environment for Laravel, PostgreSQL and PGroonga
```